luda-adventure/export-known-ids-macro/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+use lazy_static::lazy_static;
+use proc_macro::TokenStream;
+use quote::quote;
+use regex::bytes::Regex;
+use std::collections::BTreeMap;
+use syn::{
+    parse::Parse, parse_macro_input, punctuated::Punctuated, token::Colon, Ident, LitStr, Token,
+};
+
+#[proc_macro]
+pub fn export_known_ids(input: TokenStream) -> TokenStream {
+    let input = parse_macro_input!(input as MacroInput);
+
+    let exports = input.uuid_name_map.into_iter().map(|(uuid, name)| {
+        quote! {
+            pub const #name: namui::Uuid = namui::uuid!(#uuid);
+        }
+    });
+
+    TokenStream::from(quote! {
+        use namui::prelude::uuid;
+
+        #(#exports)*
+    })
+}
+
```

Comment:
This macro checks duplicated uuid and emit error at compile time.
This prevents developers from making mistakes. So I think I need.

Reply:
I think you can just make a new const uuid in your code and use it. can't it?

Reply:
Because you don't think it's going to happen that the uuid duplicated?

Reply:
Yes and I'm not sure saving data in code.

Reply:
The only way to make a couple of duplicated uuids is when you write uuid by your brain and hands. If you use uuid generator, such any website googled, you always make a non-duplicated uuid. So that mean you don't need to spend your time for manage this macro.

Reply:
`manage`... got it. let's remove this.

Reply:
[db487dc](https://github.com/NamseEnt/namseent/pull/359/commits/db487dc01071d200c454b7ee00140a9ee7e493cf)